examples/temperature-measurement-app/esp32/main/gen/af-structs.h

```diff
@@ -1,4 +1,5 @@
-/**
```

Comment:
You should probably not take it on faith.

If you look at it twice you may noticed that there are some empty structs. For example:

With `ZAP` templates:
```
// Struct for Signature
typedef struct _Signature
{
} EmberAfSignature;
```

With `AppBuilder`:
```
// Void typedef for EphemeralData which is empty.
// this will result in all the references to the data being as uint8_t*
typedef uint8_t EphemeralData;
```

It does work because those are not used. I will open an issue for them.

Reply:
#3873